src/main/resources/com/mathworks/ci/MatlabBuilder/runMatlabTests.m

```diff
@@ -111,9 +111,12 @@
     end
 end
 
-% Produce PDF test report (Not supported below R2016b)
+% Produce PDF test report (Not supported on Mac OS platform and below R2017a)
 if producePDFReport
-    if ~testReportPluginPresent
+    if ismac
+        warning('MATLAB:testArtifact:unSupportedPlatform', ...
+            'Producing a PDF test report is not supported on Mac OS platform.');
```

Comment:
on MacOS platforms.

Reply:
be sure to change both the comment and the warning text